addon/components/fixed-table-columns.js

```diff
@@ -35,6 +35,9 @@ export default TableColumns.extend({
   },
 
   actions: {
+    didRenderTable() {
+      this.sendAction('didRenderTable', this);
```

Comment:
since this can come from either `fixed-table-columns` or `table-columns`, we should probably use `run.scheduleOnce`. Right now I think its fine but I want to be safe about it.